### PR TITLE
Styling messaging

### PR DESCRIPTION
**Pull request by tsbolty**
### What this PR does (required):
- Completed styling of "dashboard" page. All messaging and conversations have been styled mostly according to the mock

### Screenshots / Videos (required):
image

image

### Any information needed to test this feature (required):
- Just send messages to people

### Any issues with the current functionality (optional):
- No known issues

### CODE REVIEW

File: client/src/components/OtherUserBubble/OtherUserBubble.tsx [hunk 1]
```diff
@@ -10,21 +10,21 @@ interface Props {
...
-		<Box className={classes.root}>
+const OtherUserBubble = ({ time, text, otherUsername, otherPhoto }: Props): JSX.Element => {
+  const classes = useStyles();
+  return (
+    <Box display="flex" flex-direction="column" alignItems="flex-end" maxWidth="90%">
       <Avatar alt={otherUsername} src={otherPhoto} className={classes.avatar}></Avatar>
-      <Box>
-        <Typography className={classes.usernameDate}>
+      <Box maxWidth="50%">
+        {/* <Typography className={classes.usernameDate}>
```
Comment (rajivtitus) [inline client/src/components/OtherUserBubble/OtherUserBubble.tsx]: Please remove commented code

File: client/src/components/SenderBubble/SenderBubble.tsx [hunk 2]
```diff
@@ -1,29 +1,27 @@
...
-			)} */}
-		</Box>
-	);
-}
+const SenderBubble = ({ time, text }: Props): JSX.Element => {
+  const classes = useStyles();
+  return (
+    <Box display="flex" justifyContent="flex-end">
+      <Box width="50%" maxWidth="45%" display="flex" justifyContent="flex-end">
+        {/* <Typography className={classes.date}>{time}</Typography> */}
```
Comment (rajivtitus) [inline client/src/components/SenderBubble/SenderBubble.tsx]: Please remove commented code/uncomment it

File: client/src/components/ActiveChat/ActiveChat.tsx [hunk 3]
```diff
@@ -18,7 +18,11 @@ export default function Dashboard(): JSX.Element {
   const classes = useStyles();
   const { convo } = useConvoContext();
   const { loggedInUser } = useAuth();
-  const [otherUser, setOtherUser] = useState<OtherUser>();
+  const [otherUser, setOtherUser] = useState<OtherUser | undefined>({
```
Comment (rajivtitus) [inline client/src/components/ActiveChat/ActiveChat.tsx]: If you have a mock user object as default state, why do we also have undefined as a type? :)

**Review by rajivtitus — APPROVED** [review]
LGTM! UI has received a big upgrade from the last time around 😄 

Just a couple of small comments, please have a look

File: client/src/components/OtherUserBubble/OtherUserBubble.tsx (hunk 1 above)
Comment (tsbolty) [reply client/src/components/OtherUserBubble/OtherUserBubble.tsx]: This is fixed.

File: client/src/components/SenderBubble/SenderBubble.tsx (hunk 2 above)
Comment (tsbolty) [reply client/src/components/SenderBubble/SenderBubble.tsx]: Also fixed.

File: client/src/components/ActiveChat/ActiveChat.tsx (hunk 3 above)
Comment (tsbolty) [reply client/src/components/ActiveChat/ActiveChat.tsx]: This is fixed.

**Pull request merged**